build/dist

```diff
@@ -192,6 +192,14 @@ cp -r "$KYUUBI_HOME/kyuubi-assembly/target/scala-$SCALA_VERSION/jars/" "$DISTDIR
 # Copy kyuubi beeline jars
 cp "$KYUUBI_HOME"/kyuubi-hive-beeline/target/*.jar "$DISTDIR/beeline-jars/"
 
+# Share the jars between server and beeline to reduce binary size
+for jar in $(ls "$DISTDIR/jars/"); do
+  if [[ -f "$DISTDIR/beeline-jars/$jar" ]]; then
+    rm "$DISTDIR/beeline-jars/$jar"
+    ln -sn "$DISTDIR/jars/$jar" "$DISTDIR/beeline-jars/$jar"
```

Comment:
Does the absolute path work if user untar to another folder?

Reply:
nice catch